recipes/quantwiz-iq/build.sh

```diff
@@ -0,0 +1,18 @@
+mkdir -p $PREFIX/bin
+module_dir=$PREFIX/MODULES
+mkdir -p $module_dir
+
```

Comment:
# Fix shebangs so perl found in environment
sed -i.bak 's|#!/usr/bin/perl|#!/usr/bin/env perl|' QuantWiz_IQ.pl